docs/src/main/paradox/geode.md

```diff
@@ -26,7 +26,7 @@ Basically it can store data as key, value. Key and value must be serialized, mor
 ##Connection
 
 First of all you need to connect to the geode cache. In a client application, connection is handle by a
- [ClientCache](https://geode.apache.org/docs/guide/14/basic_config/the_cache/managing_a_client_cache.html). A single
+ [ClientCache](https://geode.apache.org/docs/guide/15/basic_config/the_cache/managing_a_client_cache.html). A single
```

Comment:
Maybe https://developer.lightbend.com/docs/paradox/latest/features/linking.html#extref-directive would make this easier in the future?